client/gefyra/api/run.py

```diff
@@ -151,9 +158,15 @@ def run(
     if detach:
         return True
     else:
-        #
-        # 3. print out logs if not detached
-        #
-        logger.debug("Now printing out logs")
-        for logline in container.logs(stream=True):
-            print(logline.decode("utf-8"), end="")
+        try:
+            logger.debug("Now printing out logs")
+            t = Thread(target=print_logs, args=[container], daemon=True)
+            t.start()
+            while True:
+                line = sys.stdin.readline()
+                if not line:
+                    logger.info(f"Detached from container: {name}")
+                    exit(0)
```

Comment:
I'd rather not exit here, better return.

Reply:
Done!